### PR TITLE
GRPC V2 : Get Pool Delegators

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for GRPC V2 : Get Pool Delegators

## Changes
Added support for GRPC V2 : Get Pool Delegators

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetPoolDelegators", mixinStandardHelpOptions = true)
+public class GetPoolDelegators implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20002")
```
Comment (abizjak) [inline concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java:22]: Please use 20000

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    private String endpoint;
+
+    @Option(
+            names = {"--bakerId"},
+            description = "Baker Id",
+            defaultValue = "2924")
```
Comment (abizjak) [inline concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java:28]: There should not be a default for this value. There is no sensible default.

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPoolDelegatorsTest.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.delegatesTo;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientV2GetPoolDelegatorsTest {
```
Comment (abizjak) [inline concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPoolDelegatorsTest.java:33]: It'd be useful to provide a short summary of the test and its intention.

**Review by abizjak — APPROVED** [review]

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java (hunk 1 above)
Comment (parv0888) [reply concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java:22]: done

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java (hunk 2 above)
Comment (parv0888) [reply concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java:28]: done. The idea of a default value here was to enable the developers checking out this repo to just use their IDE to execute an example.

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPoolDelegatorsTest.java (hunk 3 above)
Comment (parv0888) [reply concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPoolDelegatorsTest.java:33]: done

**Pull request merged**